Utils/DynamicMethodDefinition.cs

```diff
@@ -206,8 +206,8 @@ public MethodInfo Generate()
                     if (_PreferCecil)
                         return DMDCecilGenerator.Generate(this, context);
 
-                    if (Debug)
 #if NETSTANDARD
+                    if (Debug)
```

Comment:
Maybe we can wrap this in our LUNA define?
It will be nice to see original execution flow

Reply:
Done.